extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/registry/json/JsonDistributionSummary.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.micrometer.runtime.registry.json;
+
+import java.util.Arrays;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.DoubleAdder;
+
+import io.micrometer.core.instrument.AbstractDistributionSummary;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.Measurement;
+import io.micrometer.core.instrument.Statistic;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
+import io.micrometer.core.instrument.distribution.TimeWindowMax;
+
+public class JsonDistributionSummary extends AbstractDistributionSummary {
+
+    private final AtomicLong count;
+    private final DoubleAdder total;
+    private final TimeWindowMax max;
+    private final TimeWindowMin min;
+
+    public JsonDistributionSummary(Id id, Clock clock, DistributionStatisticConfig distributionStatisticConfig,
+            double scale, boolean supportsAggregablePercentiles) {
+        super(id, clock, distributionStatisticConfig, scale, supportsAggregablePercentiles);
+        this.count = new AtomicLong();
+        this.total = new DoubleAdder();
+        this.max = new TimeWindowMax(clock, distributionStatisticConfig);
+        this.min = new TimeWindowMin(clock, distributionStatisticConfig);
+    }
+
```

Comment:
WOOHOO! =)

Reply:
AtomicLong or LongAdder?

Reply:
I basically copied this over from Micrometer's `CumulativeDistributionSummary` and they have an `AtomicLong` there ;) but yes this is probably a good idea, will change